Microsoft.Research/ClousotMain/RegressionOutput.cs

```diff
@@ -333,7 +318,8 @@ protected MethodRegressionOutcomes(Set<string> outcomes) : base(outcomes)
         {
           var outcome = (ProofOutcome)(byte)GetNamedArgOrDefault<int, Local, Parameter, Method, Field, Property, Event, Type, Attribute, Assembly>(mdDecoder, "Outcome", attr);
 
-          string msg = (string)mdDecoder.NamedArgument("Message", attr);
+          // at some places we have "this.x" instead of "x", both should be treated equal.
```

Comment:
Since it's removed on both sides, string comparison will still consider both equal.

All this code is only used by the Clousot regression tests, it does not affect the release.
Writing a test that will falsely succeed because of this would be a real challenge (if possible at all), however we had many tests that randomly fail without this workaround, depending on the environment where they run.